api/deploy.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/stormkit-io/stormkit-cli/model"
+	"github.com/stormkit-io/stormkit-cli/stormkit"
+)
+
+// DeployByIDapi is the api string formatter for Sprintf(), first argument
+// app id, second argument deploy id
+const DeployByIDapi = "/app/%s/deploy/%s"
```

Comment:
**suggestion** WDYT about moving all endpoints under the same file? Perhaps an `endpoints.go` file. We can move all endpoints there so that we can have an easy overview of the available endpoints.

Reply:
Done [3abe368](https://github.com/stormkit-io/stormkit-cli/pull/8/commits/3abe368200f2511671f666d3640508d5429464db)